pkg/registry/core/pod/strategy_test.go

```diff
@@ -267,53 +268,68 @@ func TestCheckGracefulDelete(t *testing.T) {
 	defaultGracePeriod := int64(30)
 	tcs := []struct {
 		in          *api.Pod
+		out         *metav1.DeleteOptions
```

Comment:
this is confusing... this is the delete options passed *in*, right? would be clearer to have `deleteGracePeriod *int64` and continue constructing deleteoptions inside the test